src/shared/analytics/analytics.background.ts

```diff
@@ -250,6 +271,90 @@ function trackAppEvents({ account }: { account: Account }) {
     trackTransactionSent({ status: 'failed', errorMessage, context })
   );
 
+  emitter.on('transactionFormed', async (context) => {
+    const {
+      formState,
+      quote,
+      enoughBalance,
+      warningWasShown,
+      outputAmountColor,
+      scope,
+    } = context;
+
+    // We query fungible info for input and output assets
+    // to provide fdv_asset_sent and fdv_asset_received
+    const inputAssetData = formState.inputFungibleId
+      ? await queryFungibleInfo({
+          fungibleId: formState.inputFungibleId,
+          currency: 'usd',
+        })
+      : null;
+    const inputAsset = inputAssetData?.data?.fungible;
+    const outputAssetData = formState.outputFungibleId
+      ? await queryFungibleInfo({
+          fungibleId: formState.outputFungibleId,
+          currency: 'usd',
+        })
+      : null;
+    const outputAsset = outputAssetData?.data?.fungible;
+    invariant(inputAsset, 'Unable to fetch input asset data');
+    invariant(outputAsset, 'Unable to fetch output asset data');
+
+    const slippageOptions = formState.inputChain
+      ? getSlippageOptions({
+          chain: createChain(formState.inputChain),
+          userSlippage:
+            'slippage' in formState && formState.slippage
+              ? Number(formState.slippage)
+              : null,
+        })
+      : undefined;
```

Comment:
These calculations really do not belong here : /
I think this value should be attached to the event

Reply:
Agree. I was thinking about this too. Good to get the second option!